### PR TITLE
코드 리뷰 수정, pressedButton 생성, AuthButton 수정

### PR DESCRIPTION
**Pull request by chepchep2**

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#33** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#32** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#31** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/31?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#30** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#29** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/29?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#28** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/28?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#27** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#26** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/25?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#23** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/23?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#22** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/22?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/19?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#18** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: ios/Runner/Info.plist [hunk 1]
```diff
@@ -45,5 +45,7 @@
 	<true/>
 	<key>UIApplicationSupportsIndirectInputEvents</key>
 	<true/>
+	<key>FLTEnableImpeller</key>
```
Comment (flowkater) [inline ios/Runner/Info.plist:48]: 혹시 ios 에 뭐 추가한거 있으신가요?

File: lib/common/auth_button.dart [hunk 2]
```diff
@@ -1,14 +1,18 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
-import 'package:my_baseball_record/common/auth_color.dart';
+import 'package:my_baseball_record/common/app_color.dart';
+import 'package:my_baseball_record/common/pressed_button.dart';
 
-class AuthButton extends StatefulWidget {
+//여기서 부터 가안으로 코드 작성 해보겠습니다.
```
Comment (flowkater) [inline lib/common/auth_button.dart:6]: 주석은 삭제

File: lib/common/auth_button.dart [hunk 3]
```diff
@@ -17,88 +21,142 @@ class AuthButton extends StatefulWidget {
...
+              IconTheme(
+                  data: const IconThemeData(
+                    color: AppColor.graysBlack,
+                  ),
+                  child: icon),
+              const SizedBox(width: 8),
+              Text(
+                text,
+                style: TextStyle(
+                    fontSize: 15.0, letterSpacing: 0.2, color: textColor),
```
Comment (flowkater) [inline lib/common/auth_button.dart:55]: 여기도 TextStyle 반영 상수로 바꿀 수 있을까요?

File: lib/common/auth_button.dart [hunk 4]
```diff
@@ -17,88 +21,142 @@ class AuthButton extends StatefulWidget {
...
+              ),
+            ],
           ),
         ),
       ),
     );
   }
 }
+
+// class AuthButton extends StatefulWidget {
```
Comment (flowkater) [inline lib/common/auth_button.dart:65]: 안쓰는 코드 주석 삭제

File: lib/common/pressed_button.dart [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  State<PressedButton> createState() => _PressedButtonState();
+}
+
+class _PressedButtonState extends State<PressedButton> {
+  bool isPressed = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      // onTap: () {
```
Comment (flowkater) [inline lib/common/pressed_button.dart:29]: OnTap 이 제대로 반영이 되나요? 주석이 되어있는데

**Comment by chepchep2** [comment]
[image: 스크린샷 2024-03-12 12.42.17.png]
이 부분때매 코드 추가했습니다.

2024년 3월 12일 (화) 오후 12:40, flowkater ***@***.***>님이 작성:

**Review by flowkater — CHANGES_REQUESTED** [review]
제대로 동작하는 건지 확인이 필요하네요.

**Comment by chepchep2** [comment]
혹시 실행 안되실까요?
저는 수정 뒤에는 실행이 잘되는데 실행 안되시면 다시 한번 알아보겠습니다.

2024년 3월 12일 (화) 오후 12:43, flowkater ***@***.***>님이 작성:

**Comment by chepchep2** [comment]
### Merge activity

* **Mar 17, 8:12 PM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24).
* **Mar 17, 8:30 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24) rebased this pull request as part of a merge.
* **Mar 17, 8:31 PM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24).

**Pull request merged**